src/webcrypto/crypto.ts

```diff
@@ -1,5 +1,5 @@
 // We use WebCrypto aka globalThis.crypto, which exists in browsers and node.js 16+.
 // See utils.ts for details.
 declare const globalThis: Record<string, any> | undefined;
-export const crypto =
+export const crypto: Crypto =
```

Comment:
Remove this, it adds dependency on `dom`

Reply:
Is it also necessary to remove it in cryptoNode.js?

Reply:
yes

Reply:
Can you review again?